Source/WebCore/css/parser/CSSPropertyParserConsumer+Box.cpp

```diff
@@ -45,20 +45,35 @@ RefPtr<CSSValue> consumeMarginTrim(CSSParserTokenRange& range, CSS::PropertyPars
     // https://drafts.csswg.org/css-box/#margin-trim
 
     auto firstValue = range.peek().id();
-    if (firstValue == CSSValueBlock || firstValue == CSSValueInline || firstValue == CSSValueNone)
+    if (firstValue == CSSValueNone)
```

Comment:
Please update the grammar entry in CSSProperties.json.

Reply:
Done